### PR TITLE
Introduce verification_status in Account schema

### LINKED ISSUE #146: Support Account.verification_status
Hello!

According to the Plaid docs here, the account object should have a nullable field called verification_status, which is an enum with 4 values

pending_automatic_verification | An Item is pending automated microdeposit verficiation
automatically_verified | An Item was successfully automically verified
pending_manual_verification | An Item is pending manual microdeposit verficiation
manually_verified | An Item was successfully manually verified

https://plaid.com/docs/#account-verification-status

Any chance we can bump the library forward to include these entries in the response?

Thanks! Let me know how I can help.

### PR DESCRIPTION
**Pull request by nathankot**
Closes #146

### CODE REVIEW

File: src/main/java/com/plaid/client/response/Account.java [hunk 1]
```diff
@@ -1,9 +1,18 @@
 package com.plaid.client.response;
 
 public final class Account {
+
+  /**
+   * The verification status
+   */
+  public enum VerificationStatus {
+    pending_automatic_verification, pending_manual_verification, manually_verified
```
Comment (davidzhanghp) [inline src/main/java/com/plaid/client/response/Account.java]: oh no undercase enums

File: src/test/java/com/plaid/client/integration/AbstractItemIntegrationTest.java [hunk 2]
```diff
@@ -30,6 +30,7 @@
   protected static void assertAccount(Account actualAccount,
                                       String expectedType,
                                       String expectedSubType,
+                                      Account.VerificationStatus expectedVerificationStatus,
```
Comment (davidzhanghp) [inline src/test/java/com/plaid/client/integration/AbstractItemIntegrationTest.java]: does this not make other things fail to compile?

File: src/main/java/com/plaid/client/response/Account.java (hunk 1 above)
Comment (sloanesturz) [reply src/main/java/com/plaid/client/response/Account.java]: looks like you're missing `automatically_verified`? 

(`automatically_verified` is listed in https://plaid.com/docs/#account-verification-statuses but not in https://plaid.com/docs/#onsuccess-callback)

Comment (nathankot) [reply src/main/java/com/plaid/client/response/Account.java]: you're right @sloanesturz 🙈 just added this

File: src/test/java/com/plaid/client/integration/AbstractItemIntegrationTest.java (hunk 2 above)
Comment (nathankot) [reply src/test/java/com/plaid/client/integration/AbstractItemIntegrationTest.java]: yes it did 😅 I've updated the PR and made sure it compiles on my own machine... we should get CI in though

**Comment by nathankot** [comment]
@davidzhanghp PTAL! specifically I'm not so sure about the lowercased `VerificationStatus` status enums and how they get serialized from JSON

**Comment by davidzhanghp** [comment]
@nathankot the standard is to just treat the field a string. can we do that?

weird things happen if we declrae an enum this way and we get a value that we don't know about

File: src/main/java/com/plaid/client/response/Account.java [hunk 3]
```diff
@@ -1,9 +1,18 @@
...
+   */
+  public enum VerificationStatus {
+    pending_automatic_verification, pending_manual_verification, manually_verified, automatically_verified
+  }
+
   private String accountId;
   private String type;
   private String subtype;
+  private VerificationStatus verificationStatus;
   private Balances balances;
```
Comment (supkisids50) [inline src/main/java/com/plaid/client/response/Account.java:7]: Open

**Review by supkisids50 — APPROVED** [review]

**Pull request closed without merging**